modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -51,12 +51,55 @@ Before configuring host network interfaces, review the OpenShift Container Platf
 ----
 <1> Add NMState YAML syntax to configure host interfaces.
 
+[IMPORTANT]
+====
+During node configuration, you must create PTR records. PTR records are required for valid host name configuration.
```

Comment:
I feel you can be even more specific:
you must create PTR records -> "For each node being configured with a static IP, you must configure a DNS PTR record containing a valid hostname and the specific IP associated with that node."